introduction.md

```diff
@@ -8,3 +8,8 @@ Javascript
 Python
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?